shaders/src/light_indirect.fs

```diff
@@ -345,7 +453,7 @@ void evaluateIBL(const MaterialInputs material, const PixelParams pixel, inout v
 #if IBL_INTEGRATION == IBL_INTEGRATION_PREFILTERED_CUBEMAP
     vec3 E = specularDFG(pixel);
     vec3 r = getReflectedVector(pixel, n);
-    Fr = E * prefilteredRadiance(r, pixel.perceptualRoughness);
+    Fr = E * prefilteredRadiance(r, pixel.perceptualRoughnessUnclamped);
```

Comment:
We should use the clamper value for reflections. Now there's a mismatch between the IBL and analytical lights.

Reply:
Does it matter though? I mean, isn't it better to have sharp reflections?  I was wondering about that.

Reply:
I think it does yes. Also it helps with specular aliasing. Besides the reflections are not that blurry.

Reply:
ok, I'll put change it back.